jobboard/templates/jobboard/job.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %} {% block content %}
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md p-5 text-align">
+            <h2>{{ job_data.title }} </h2>
+            <h6> Company: {{ job_data.company }}, in {{ job_data.city }}</h6>
+            <h7>{{ job_data.job_type }} job ;</h7>
+            <h7> Work from: {{ job_data.work_from }}</h7>
+            <br> <br>
+            <h6><strong>{{ job_data.description }}</strong></h6>
+            <br>
+            {% if user.is_authenticated %}
+                {% if is_student %}
+                    {% if is_student_applied %}
+                        <p>YOU HAVE ALREADY APPLIED FOR THIS JOB</p>
+                    {% else %}
+                        <button onclick="location.href = 'apply_for_job/'" type="submit">APPLY NOW</button>
+                    {% endif %}
+                {% endif %}
+
+<!--                {% if is_recruiter %}-->
+<!--                    {% if job_data.recruiter.user.id == user.id %}-->
+<!--                        <button onclick="location.href = 'edit_job/{{ job.id }}/'" type="submit">Edit this job</button>-->
+<!--                    {% endif %}-->
+<!--                {% endif %}-->
```

Comment:
why do these lines in the comment?

Reply:
I'm waiting for the edit job page to be added. Afterwards, I'll add here the link to that page, and add tests to the render of that page. I'll push tomorrow a PR without these lines in the meantime

Reply:
@yaelerdr the edit job page won't make it. so I think you could update it in your PR. we have 9 PR's open so I don't think we will be able to add more features before the final demo